plugins/calibre.koplugin/main.lua

```diff
@@ -109,7 +109,7 @@ function Calibre:addToMainMenu(menu_items)
     -- insert the metadata search
     if G_reader_settings:isTrue("calibre_search_from_reader") or not self.ui.view then
         menu_items.find_book_in_calibre_catalog = {
-            text = _("Find a book via calibre metadata"),
+            text = _("Calibre search"),
```

Comment:
While I lean more toward "our style guide says to start a phrase or fragment with an uppercase character," I think `calibre search` would also be included in that.

@NiLuJe probably has a better finger on the calibre pulse.

Reply:
There's a bit of a catch-22 in that the official spelling of the app is actually `calibre` (lowercase), which has a tendency to drive me mildly nuts ;p.

Reply:
In any case, in this context, I feel like that would be a bit *too* short, anyway ;).

And people are already confused enough that I think the current spelling is perfectly fine?

Reply:
> In any case, in this context, I feel like that would be a bit too short, anyway ;).

You mean it should be at least `calibre metadata search` or just no change at all?

Reply:
`calibre metadata search` would work, FWIW, but I also have no issue with the current wording ;).

Reply:
I slightly prefer calibre metadata search in the sense that it's more succinct, but no real preference

Reply:
```suggestion
            text = _("Calibre metadata search"),
```